python/hsfs/feature_group.py

```diff
@@ -1071,10 +1330,19 @@ def insert(
 
         !!! example "Upsert new feature data with time travel format `HUDI`:"
             ```python
-            fs = conn.get_feature_store();
-            fg = fs.get_feature_group("example_feature_group", 1)
-            upsert_df = ...
-            fg.insert(upsert_df)
+            # connect to the Feature Store
+            fs = ...
+
+            fg = fs.get_or_create_feature_group(
+                name='bitcoin_price',
+                description='Bitcoin price aggregated for days',
+                version=1,
+                primary_key=['unix'],
+                online_enabled=True,
+                event_time=['unix']
+            )
+
+            fg.insert(df_bitcoin_processed)
```

Comment:
show example and explain the difference with `fg.insert(fraud_labels, write_options={"wait_for_job" : False})`